include/aws/crt/auth/Credentials.h

```diff
@@ -125,7 +125,12 @@ namespace Aws
              */
             struct AWS_CRT_CPP_API CredentialsProviderStaticConfig
             {
-                CredentialsProviderStaticConfig() : AccessKeyId{}, SecretAccessKey{}, SessionToken{} {}
+                CredentialsProviderStaticConfig()
+                {
+                    AWS_ZERO_STRUCT(AccessKeyId);
+                    AWS_ZERO_STRUCT(SecretAccessKey);
+                    AWS_ZERO_STRUCT(SessionToken);
+                }
 
                 /**
                  * The value of the access key component for the provider's static aws credentials
```

Comment:
This was somehow pulled in from merged commit: https://github.com/awslabs/aws-crt-cpp/commit/8e4da9be463cd74d1a9f9f7ff4f113f3b85c4c9a